corehq/messaging/smsbackends/amazon_pinpoint/models.py

```diff
@@ -57,8 +58,13 @@ def send(self, msg, *args, **kwargs):
                 }
             }
         }
-        response = client.send_messages(
-            ApplicationId=config.project_id,
-            MessageRequest=message_request
-        )
-        return response
+        try:
+            response = client.send_messages(
+                ApplicationId=config.project_id,
+                MessageRequest=message_request
+            )
+            msg.backend_message_id = response['MessageResponse']['Result'][phone_number]['MessageId']
+            msg.save()
```

Comment:
Save not necessary because it [will be called later](https://github.com/dimagi/commcare-hq/blob/ac7aa4c23fde62339b6506fc94bdf9f81e865971/corehq/apps/sms/api.py#L319).
```suggestion
```